tools/pot/tests/test_graph.py

```diff
@@ -23,7 +23,7 @@
     ('googlenet_example', 'pytorch', 'ANY'),
     ('mobilenetv2_ssd_example', 'pytorch', 'ANY'),
     ('densenet121_example', 'pytorch', 'ANY'),
-    ('multiple_out_ports_net', 'tf', 'ANY'),
+    # ('multiple_out_ports_net', 'tf', 'ANY'),
```

Comment:
I would recommend to tag it somehow, e.g. with the comment "TODOL CI", in order to identify such gaps easily in the future.